app/build.gradle

```diff
@@ -88,6 +88,7 @@ dependencies {
 
     //User Item
     implementation 'com.google.android:flexbox:0.3.0-alpha2'
+    implementation 'com.google.firebase:firebase-firestore-ktx:21.6.0'
```

Comment:
is it OK to ask you to move it under 'firebase' section?

Reply:
ok